src/Routers/FunctionsRouter.js

```diff
@@ -26,19 +26,19 @@ export class FunctionsRouter extends PromiseRouter {
       }
     }
   }
-  
+
   static handleCloudFunction(req) {
     var applicationId = req.config.applicationId;
     var theFunction = triggers.getFunction(req.params.functionName, applicationId);
     var theValidator = triggers.getValidator(req.params.functionName, applicationId);
     if (theFunction) {
-
       const params = Object.assign({}, req.body, req.query);
       var request = {
         params: params,
         master: req.auth && req.auth.isMaster,
         user: req.auth && req.auth.user,
-        installationId: req.info.installationId
+        installationId: req.info.installationId,
+        log: req.config.loggerController && req.config.loggerController.adapter
```

Comment:
I've thought about this a bit more and I have thought of a good reason for `LoggerAdapter` to have a consistent interface: If any other adapter (such as DB adapter) want to log stuff, we can give them a logger so that they can do that. Perhaps we should rethink this.